pkg/netns/netns_linux.go

```diff
@@ -51,13 +52,24 @@ func GetNSRunDir() (string, error) {
 // NewNS creates a new persistent (bind-mounted) network namespace and returns
 // an object representing that namespace, without switching to it.
 func NewNS() (ns.NetNS, error) {
-	b := make([]byte, 16)
-	_, err := rand.Reader.Read(b)
-	if err != nil {
-		return nil, fmt.Errorf("failed to generate random netns name: %v", err)
+	for i := 0; i < 1000000; i++ {
```

Comment:
Maybe a little lower on the iteration count here? 10,000 or so?

Reply:
Yeah honestly not sure what a good number is, 10k would be fine for me as well.